_data/banking.yml

```
@@ -246,9 +246,12 @@ websites:
 
   - name: CIBC
     url: https://www.cibc.ca
```

```suggestion
    url: https://www.cibc.com
```
Permanent redirect, even from Canadian IPs.